thunder/core/utils.py

```diff
@@ -1123,3 +1124,13 @@ def partition(pred, iterable):
     # partition(is_odd, range(10)) --> 0 2 4 6 8   and  1 3 5 7 9
     t1, t2 = itertools.tee(iterable)
     return itertools.filterfalse(pred, t1), filter(pred, t2)
+
+
+def make_hashable(item: Any, /) -> tuple:
+    if isinstance(item, (list, tuple)):
```

Comment:
Is there anything useful in collections.abc that would cover lists, tuples, and all other similar objects?

Reply:
Actively looking out for that actually! I would be such a neater solution, do you have any pointer?

Reply:
Sequence